src/components/AspectRatio.web.tsx

```diff
@@ -1,50 +1,40 @@
 import React from "react";
 import {
   View,
-  createElement,
   StyleSheet,
-  LayoutChangeEvent,
+  LayoutRectangle,
   StyleProp,
   ViewStyle,
 } from "react-native";
 
 interface Props {
   children?: React.ReactNode;
-  onLayout?: (event: LayoutChangeEvent) => void;
-  ratio: number;
```

Comment:
i think we need ratio here still, no?

Reply:
It uses ratio directly provided in the style, no need for additional prop